astropy/coordinates/representation.py

```diff
@@ -2227,6 +2278,24 @@ def norm(self):
         """
         return np.abs(self.r)
 
+    def _scale_operation(self, op, *args):
```

Comment:
This is the same as SphericalRep. Out of Scope, but in future could we combine the two with a SphericalBase and just use .components to agnostically refer to the angular versus radial pieces?

Reply:
Wouldn't quite work, because for the colatitude one cannot just change the sign. Still, it does feel like something more could be factored out...

Reply:
Like in my transform code, as @adrn pointed out, there was a fair bit of overlap between the spherical classes. I took a stab at it, but generalizing over the operations on the coordinates for Spherical and PhysicsSpherical was a little prohibitive.

Reply:
I do agree...